ray_julia_jll/deps/wrapper.cc

```diff
@@ -475,7 +476,11 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
     mod.add_type<ObjectID>("ObjectID")
         .method("ObjectIDFromHex", &ObjectID::FromHex)
         .method("ObjectIDFromRandom", &ObjectID::FromRandom)
-        .method("ObjectIDFromNil", &ObjectID::Nil)
+        .method("ObjectIDFromNil", []() {
+            auto id = ObjectID::Nil();
+            ObjectID id_deref = id;
+            return id_deref;
+        })
```

Comment:
Should add a test for this

Reply:
what's going on here exactly?

Reply:
`ObjectID::Nil` returns a ref to an `ObjectID` (unlike all the otehr `FromX` functions).  that means taht when comparing two objectIDs we need to handle BOTH values AND refs, and that was even more annoying than I epxected (was getting weird method errors even using `@cxxdereference` because of _slightly_ less generic fallbacks generated by CxxWrap itself).  So I just decided to punt here and return a value even though it's a bit less efficient.

Reply:
ah I see - thanks!

Reply:
Still needs a test

Reply:
it'll eventually be covered by the tests for teh ownership registration (there weren't any other uses of it or pre-existing tests)

Reply:
...and none of the other `FromX` have direct tests either